lib/src/vector_math/vector3.dart

```diff
@@ -555,46 +555,46 @@ class Vector3 implements Vector {
   }
 
   set xy(Vector2 arg) {
-    final argStorage = arg._v2storage;
-    _v3storage[0] = argStorage[0];
+    final argStorage = arg.storage;
     _v3storage[1] = argStorage[1];
+    _v3storage[0] = argStorage[0];
   }
 
   set xz(Vector2 arg) {
-    final argStorage = arg._v2storage;
-    _v3storage[0] = argStorage[0];
+    final argStorage = arg.storage;
```

Comment:
please restore the ._v2storage accessor here

Reply:
Ah, was sure I had set all of them back, found some for the matrix too.
They should all be fixed now!

Reply:
Oops, shouldn't have done that in vim, one more fix coming up!

Reply:
Now it should be fine